Configure

```diff
@@ -11,6 +11,7 @@
 
 use 5.10.0;
 use strict;
+use warnings;
 use Config;
```

Comment:
I would not turn on warnings without back-checking a wide range of perl versions ...